iceoryx_posh/test/moduletests/test_capro_service.cpp

```diff
@@ -114,32 +137,69 @@ TEST_F(ServiceDescription_test, CtorNoParams)
 
 TEST_F(ServiceDescription_test, CtorIDStrings)
```

Comment:
Please create multiple tests here and rename them so that the outcome is clear to the reviewer. Something like `StringCTorCreatesServiceDescriptionWithValidClassHash`.